src/pages/library/datasetBuilder/CohortEditor.ts

```diff
@@ -1,8 +1,12 @@
+import 'rc-slider/assets/index.css';
```

Comment:
It would be nice to avoid either having to duplicate this import everywhere that `rc-slider` is used or relying on the side-effect of this module importing it. Could this import be moved to `appLoader.js` or maybe have a module in components/common that imports this CSS and also re-exports `rc-slider`?

Reply:
I think a module would be good-- that's what we do with the Highcharts chart, which also enables us to enable a couple of modules.